composer.json

```diff
@@ -14,7 +14,11 @@
         "vimeo/psalm": "^3.0.2"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^3.3"
+        "codeception/base": "^2.5",
+        "muglug/package-versions-56": "^1.2",
+        "squizlabs/php_codesniffer": "^3.3.1",
```

Comment:
I bumped the version number slightly here because `3.3.0` has a `continue` in a switch which causes a Notice in PHP 7.3, which causes the "lowest" tests to fail on that version of PHP (`3.3.1` fixed the issue: https://github.com/squizlabs/PHP_CodeSniffer/commit/16c771d0ac0b4c3e425a4f9c9533d96832e0f08d)